examples/model.py

```diff
@@ -190,9 +190,10 @@ class Meta:
 ]))
 
 # Update list attribute
+thread_item.notes.append(["new note"])
 print(thread_item.update(actions=[
     Thread.notes.set(
-        Thread.notes.append(["new note"])
+        thread_item.notes
     )
```

Comment:
It's confusing, but the code you've changed is how you add an item to a list atomically in DynamoDB.

In "raw" DynamoDB, that'd be
```
SET Thread.notes = list_append(Thread.notes, "new note")
```
See [docs](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_UpdateItem.html#DDB-UpdateItem-request-UpdateExpression)